### ISSUE TITLE
Missing argument "sensitive = true" in public doc

### ISSUE THREAD

**Issue by ishanslab** [hashicorp/terraform-provider-azurerm#15154]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version  
Terraform v1.1.4
on windows_amd64
provider registry.terraform.io/hashicorp/azurerm v2.93.1

### Affected Resource(s)

* `azurerm_key_vault_secret`
* `azurerm_key_vault_secrets`

### Terraform Configuration Files

Missing argument "sensitive = true" in public doc.
```hcl
data "azurerm_key_vault_secret" "example" {
  name         = "tfkey"
  key_vault_id = data.azurerm_key_vault.example.id
}

output "secret_value" {
  value = data.azurerm_key_vault_secret.example.value

}

```

### Expected Behaviour

 + secret_value = (sensitive value)

### Actual Behaviour

│ If you do intend to export this data, annotate the output value as
│ sensitive by adding the following argument:
│     sensitive = true

### Steps to Reproduce

[use this to reproduce the issue](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)

1. `terraform plan`

### References

* [Data Source: azurerm_key_vault_secret](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)
* [Data Source: azurerm_key_vault_secrets](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secrets)

**Comment by sinbai** [comment]
@ishanslab thanks for reporting this issue. The fix has been submitted.

**Comment by stephybun** [comment]
Closed by #15160
